config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-0.yaml

```diff
@@ -139,7 +139,7 @@ periodics:
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
 - name: periodic-cluster-api-verify-book-links-release-1-0
-  interval: 24h
+  interval: 12h
```

Comment:
Sorry same here, let's keep it at 24h

Reply:
Updated these changes, as well as did a rebase.